plugins/versionpress/tests/Unit/QueryLanguageUtilsTest.php

```diff
@@ -185,12 +185,14 @@ public function rulesAndGitLogQueryProvider()
                 '-i --all-match --grep="^VP-Action: \(entity\|.*\)/.*/\(.*vp.*\|vpid\)$"'
             ],
             [['text' => ['text1', 'Test text', '*']], '-i --all-match --grep="text1" --grep="Test text" --grep=".*"'],
+
             [['x-vp-another-key' => ['Test value']], '-i --all-match --grep="^x-vp-another-key: \(Test value\)$"'],
-            [['vp-another-key' => ['Test value']], '-i --all-match --grep="^\(x-\)\?vp-another-key: \(Test value\)$"'],
-            [['another-key' => ['Test value']], '-i --all-match --grep="^\(x-vp-\|vp-\)another-key: \(Test value\)$"'],
+            [['X-VP-another-key' => ['Test value']], '-i --all-match --grep="^X-VP-another-key: \(Test value\)$"'],
+            [['vp-another-key' => ['Test value']], '-i --all-match --grep="^\(X-\)\?vp-another-key: \(Test value\)$"'],
```

Comment:
All these tests now verify the following behavior, implemented in 244daf4f3d2fd88dd051943b96fe140f963d121b:

- Prefix _detection_, i.e., whether a field already contains `X-VP-` or `VP-`, is case insensitive. 
- But, for actually producing the Git search string, there is no active conversion:
    - VersionPress always adds `(X-)VP-` (if necessary) in upper case.
    - User input is untouched.

This can lead to mixed casing like `X-VP-some-key` which is fine – the `-i` flag won't care.